### PR TITLE
Add WebView versions for WorkerGlobalScope API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for WebView Android for the `WorkerGlobalScope` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.8).  The collector obtains results based upon the latest WebView version (to determine if it is supported), then version numbers are copied from Chrome Android.  Results are manually confirmed for accuracy.

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/WorkerGlobalScope

### CODE REVIEW

File: api/WorkerGlobalScope.json [hunk 1]
```diff
@@ -666,7 +666,7 @@
               "version_added": "4.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": false
```
Comment (foolip) [inline api/WorkerGlobalScope.json:669]: Why false here? It’s in Chrome for Android...

File: api/WorkerGlobalScope.json [hunk 2]
```diff
@@ -714,7 +714,7 @@
...
```
Comment (foolip) [inline api/WorkerGlobalScope.json:717]: Same here.

File: api/WorkerGlobalScope.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/WorkerGlobalScope.json:669]: Looks like this is a case of a bad test in the collector (I'm surprised that the rest of the tests worked properly).  I'll have a fix for this in collector v1.1.9; reverting and adding version numbers.

Comment (foolip) [reply api/WorkerGlobalScope.json]: Great, thanks!

**Review by foolip — APPROVED** [review]

**Pull request merged**